### PR TITLE
Fix nightly

### PR DESCRIPTION
**Pull request by vmuzikar**
Closes https://github.com/keycloak-rel/keycloak-rel/issues/42

Supersedes #43.

It's not a super nice solution but if we want to avoid [external dependencies](https://github.com/andymckay/cancel-action/tree/0.2), this is the most straightforward way.

### CODE REVIEW

File: .github/workflows/release-nightly.yml [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+  schedule:
+    - cron: '0 2 * * *'
   workflow_dispatch:
 
 concurrency: rel-nightly
 
 jobs:
 
   create-tags:
+    if: github.repository_owner == 'keycloak-rel'
```
Comment (jonkoops) [inline .github/workflows/release-nightly.yml:13]: This is the only place this is needed as all of the subsequent steps are dependent on this job. Thus if this job is skipped so are all others.

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: .github/workflows/release-nightly.yml (hunk 1 above)
Comment (vmuzikar) [reply .github/workflows/release-nightly.yml:13]: Good point, thank you.

**Review by jonkoops — APPROVED** [review]

**Review by abstractj — APPROVED** [review]

File: .github/workflows/release-nightly.yml (hunk 1 above)
Comment (stianst) [inline .github/workflows/release-nightly.yml:13]: Need to check if it's a scheduled event as we need to be able to run nightly flow manually on test fork

**Review by stianst — CHANGES_REQUESTED** [review]

File: .github/workflows/release-nightly.yml (hunk 1 above)
Comment (jonkoops) [inline .github/workflows/release-nightly.yml:13]:
```suggestion
    if: github.event_name != 'schedule' || github.repository_owner == 'keycloak-rel'
```

**Comment by vmuzikar** [comment]
@stianst @jonkoops Thanks for the review! Applied the suggestion.

**Review by jonkoops — APPROVED** [review]

**Review by abstractj — APPROVED** [review]

**Pull request merged**